app/assets/javascripts/constants/api_constants.es6.jsx

```diff
@@ -19,6 +19,8 @@
     get csvs() {
       return {
         students: '/api/students.csv',
+        schools: '/api/schools.csv',
+        groups: '/api/groups/1.csv',
```

Comment:
Instead of hard coding `1.csv`, why not turn this into a function that takes an `id` as a parameter?
Check out some of the other api constants for examples.